src/api.rs

```diff
@@ -357,6 +365,12 @@ impl Default for PixelRange {
     }
 }
 
+/// Contain different types of Matrix Coefficients which are used for deriving the luma and chroma (YCbCr) from the R, G, B primaries.
+/// - This is used to describe the color parameters for the video to be encoded, the default value is `Unspecified`.
+/// - Matrix Coefficients can also covert across a multitude of systems.
+/// - For more information please refer to `Table 4` of [`ISO/IEC 23008-9's ITU H.273 Standard`].
+///
+/// [`ISO/IEC 23008-9's ITU H.273 Standard`]:  https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-H.273-201612-I!!PDF-E&type=items
```

Comment:
does this really need a link every time?

Reply:
Okay, WIll remove it